Tests/TestObjectTypes/ObjectTypesTests.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+
+final class ObjectTypesTests: MockoloTestCase {
+
+    func testObjectTypesWithNoProperty() {
+        verify(srcContent: argumentsObjectTypesWithNoAnnotation,
+               dstContent: argumentsObjectTypesWithClassAnnotationMock)
+    }
+
+    func testObjectTypesWithClassProperty() {
+        verify(srcContent: argumentsObjectTypesWithClassAnnotation,
+               dstContent: argumentsObjectTypesWithClassAnnotationMock)
+    }
+
+    func testObjectTypesWithActorProperty() {
+        verify(srcContent: argumentsObjectTypesWithActorAnnotation,
+               dstContent: argumentsObjectTypesWithActorAnnotationMock)
+    }
+}
```

Comment:
Supports Swift < 5.5.

```suggestion
#if compiler(>=5.5.2) && canImport(_Concurrency)
final class ObjectTypesTests: MockoloTestCase {

    func testObjectTypesWithNoProperty() {
        verify(srcContent: argumentsObjectTypesWithNoAnnotation,
               dstContent: argumentsObjectTypesWithClassAnnotationMock)
    }

    func testObjectTypesWithClassProperty() {
        verify(srcContent: argumentsObjectTypesWithClassAnnotation,
               dstContent: argumentsObjectTypesWithClassAnnotationMock)
    }

    func testObjectTypesWithActorProperty() {
        verify(srcContent: argumentsObjectTypesWithActorAnnotation,
               dstContent: argumentsObjectTypesWithActorAnnotationMock)
    }
}
#endif
```

Reply:
or use `XCTSkip`

Reply:
I prefer to skip the test, but I'm okay with not including it in the build.
https://github.com/uber/mockolo/pull/222#issuecomment-1465644279